### ISSUE TITLE
Ability to removeFromQueue 

### ISSUE THREAD

**Issue by loria33** [react-native-google-cast/react-native-google-cast#378]
Is there an option to remove from the cast queue?
Queue items only has 3 items when retrieving from getMediaStatus , I think this is a limitation . But Cast SDK has a remove items function by itemID.

RN 0.63
Google Cast : v4

**Comment by petrbela** [comment]
Right, that function is currently missing. Feel free to add it and open a PR.
